provider/core/src/helpers.rs

```diff
@@ -76,3 +76,70 @@ fn test_escape_for_json() {
         escape_for_json("ab\u{001F}c", &mut String::new())
     );
 }
+
+/// Const function to compute the FxHash of a byte array with little-endian byte order.
+#[allow(dead_code)]
+pub const fn fxhash_32(bytes: &[u8]) -> u32 {
```

Comment:
Please add a comment why you chose this hash function.

Reply:
Done.